src/main/webapp/app/stores/open-comment.store.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash';
+import { action, makeObservable, observable } from 'mobx';
+class CommentsToDeleteStore {
+  commentIndiciesToDelete: number[] = [];
+
+  constructor() {
+    makeObservable(this, {
+      commentIndiciesToDelete: observable,
+      addCommentIndex: action.bound,
+      removeCommentIndex: action.bound,
+      clear: action.bound,
+    });
+  }
+
+  addCommentIndex(index: number) {
+    this.commentIndiciesToDelete.push(index);
+  }
+
+  removeCommentIndex(index: number) {
+    _.remove(this.commentIndiciesToDelete, commentIndex => commentIndex === index);
+  }
+
+  clear() {
+    this.commentIndiciesToDelete = [];
+  }
+}
+
+export const openCommentScrollPosition = observable.box<number>(null);
+export const openCommentsIdStore = observable.box<string>(null);
+export const commentsToDeleteStore = new CommentsToDeleteStore();
```

Comment:
I think you can initiate the store in the `createStore.ts` folder where all our stores are defined

Reply:
If you instantiate this store in the createStore, then you can directly inject this store into the component that you need to use the data in.

You can use the same approach as described in my comment about VusTable